_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -411,25 +437,27 @@ GET _plugins/_transform/
 }
 ```
 
-You can specify these options as the `GET` API operation’s URL parameters to filter results:
+### Query parameters
+
+You can specify these options as the GET API operation’s URL parameters to filter results:
```

Comment:
Replace the colon with a period. Can we make the sentence any clearer? Maybe something like "You can use the GET API operation's URL parameters to filter results"? Should there be a "the" before "results" ("...to filter the results")?